src/schema.sql

```diff
@@ -0,0 +1,12 @@
+create table if not exists settings (
+  name text unique not null check (name <> ""),
```

Comment:
This probably deserves an index

Reply:
yes, especially with getActorInfo on every page load?